internal/persistence/postgres/postgres.go

```diff
@@ -37,6 +38,7 @@ type Opts struct {
 	Password         string
 	EnableTLS        bool
 	CABundleFSPath   string
+	SslMode          string
```

Comment:
Should this be an enumeration or will an error occur is the `sslmode` is invalid for Postgres?

Reply:
yes, an error is returned by postgres: https://github.com/Kong/koko/pull/494#issue-1422502882

that's why I have not added an enum and did any koko validation. That option is only passed to the postgresql driver, so there is not much value doing so unless we really want to catch the config error at **config loading** instead of on dsn parsing+connection (do we?)